adsb/README.md

```diff
@@ -0,0 +1,126 @@
+| station_latitude  | string | -              | The latitude of where the RTL-SDR is located. This is used to calculate aircraft position.  | yes      |
+| station_longitude | string | -              | The longitude of where the RTL-SDR is located. This is used to calculate aircraft position. | yes      |
+| station_name      | string | Generated UUID | Simple name for displaying on map UI.                                                       | no       |
+| dump1090_ip       | string | -              | Address of dump1090 telnet server                                                           | yes      |
+| dump1090_port     | int    | 30002          | Port of dump1090 telnet server                                                              | no       |
+
+Once a link is made to the Processor Actor, contacts will start being processed.
+
+### Processor Actor 
+
+This is a simple actor that takes `Adsb.HandleAdsbMsg` messages from a dump1090 provider, creates JSON data types, and stores it in a keyvalue store.  Services like the API Gateway and UI use the keyvalue store to access the data.
+
+Each contact and station is stored with its ICAO number and UUID, respectively.  All keys are stored with a 10minute TTL to ensure stall contacts eventually fall off.
+
+### API Gateway Actor 
+
+The API Gateway actor is a way to access the ADSB data via http queries.  
+
+| Endpoint  | Method | Description                                                                                             |
+| --------  | ------ | -----------                                                                                             |
+| /contacts | GET    | Returns the air contacts as identified by all stations                                                  |
+| /stations | GET    | Returns a list of stations and their location                                                           |
+| /geojson  | GET    | Returns a combination contacts and stations in [geojson](https://www.rfc-editor.org/rfc/rfc7946) format |
+
+Any other query to the API should return a 404 - `{"error":"invalid_request"}`
+
+### UI Actor (Vue3 App)
+
+The UI actor embeds a Vue3 webapp that is used to plot contacts and stations on a map for a visual.  It utilizes the API Gateway as its datasource.
+
+### ADSB Interface
+
+## Build Instructions 
+
+> You will need [dump1090](https://github.com/wasmcloud/wasm-air) and [wash](https://github.com/wasmcloud/wash) installed.  
+
+#### Build the interface
+
+From the root of the repository, run 
+
+```
+wash gen
+```
+
+this will drop generated files in the provider and processer repositories.  Without running this generation, the rest of the components will fail to build.
+
+#### Build the provider 
+
+###### Build the Go binary
+```
+cd adsb-provider
+```
+
+We have included a [goreleaser]() configuration file to make this step easy.  If you have cli installed, simple run 
+
+```
+goreleaser build --clean --snapshot
```

Comment:
This is failing for me:
```
goreleaser build --clean --snapshot
  • starting build...
  • loading config file                              file=.goreleaser.yaml
  • loading environment variables
  • getting and validating git state
    • building...                                    commit=7773961207e138e0417dd295b3c00fdf9733c5d3 latest tag=blobby-v0.2.0
    • pipe skipped                                   reason=disabled during snapshot mode
  • parsing tag
  ⨯ build failed after 0s                    error=failed to parse tag 'blobby-v0.2.0' as semver: Invalid Semantic Version```

Reply:
this is a tough one.  since this repo has multiple tags for multiple projects, goreleaser can't differentiate.  

however, this specific error is because we dont use SEMVER compliant tags 😞